dotnet/src/Experimental/Agents/Models/ThreadMessageModel.cs

```diff
@@ -118,6 +118,69 @@ public sealed class TextContentModel
         /// Any annotations on the text.
         /// </summary>
         [JsonPropertyName("annotations")]
-        public List<object> Annotations { get; set; } = new List<object>();
+        public List<TextAnnotationModel> Annotations { get; set; } = new List<TextAnnotationModel>();
+    }
+
+    public sealed class TextAnnotationModel
+    {
+        /// <summary>
+        /// Type of content.
+        /// </summary>
+        [JsonPropertyName("type")]
+        public string Type { get; set; } = string.Empty;
+
+        /// <summary>
+        /// The text in the message content that needs to be replaced.
```

Comment:
That comment seems like a copy/paste error

Reply:
This is directly from OpenAI docs...I can clarify, however.